server/js/topics.js

```diff
@@ -0,0 +1,77 @@
+class DAS {
```

Comment:
We could use TS here with enums that better reflect the purpose of these classes.

Reply:
I thought about using TS here but decided setting up TS for the server was beyond the scope of this PR. Curious, why do you think this would be better represented by Enums? I feel like enums should be used to enumerate a finite set of cases, but our list of MQTT topics isn't necessary complete.

Reply:
I was under the impression that our list of MQTT topics would be complete. I do agree that it's not worth setting up TS for the server though.